Sources/VergeStore/Store/Derived.swift

```diff
@@ -130,7 +130,7 @@ public class Derived<Value>: _VergeObservableObjectBase, DerivedType {
         break
       case .updated(let newState):
         store?.commit {
-          $0 = newState
+          $0.replace(with: newState)
```

Comment:
The reason why we use this method to replace value is we can't override `=` operator.